buildSrc/src/main/java/com/squareup/workflow1/buildsrc/artifacts/ArtifactsTask.kt

```diff
@@ -52,27 +54,34 @@ abstract class ArtifactsTask(
   private fun Project.createArtifactList(): List<ArtifactConfig> {
 
     val map = subprojects
-      .mapNotNull { sub ->
+      .flatMap { sub ->
+        sub.extensions.findByType(PublishingExtension::class.java)
+          ?.publications
+          ?.filterIsInstance<MavenPublication>()
+          .orEmpty()
+          .map { publication -> sub to publication }
+      }
+      .mapNotNull { (sub, publication) ->
 
-        val group = sub.properties["GROUP"] as? String
-        val artifactId = sub.properties["POM_ARTIFACT_ID"] as? String
-        val pomName = sub.properties["POM_NAME"] as? String
-        val packaging = sub.properties["POM_PACKAGING"] as? String
+        val group: String? = publication.groupId
```

Comment:
Instead of reading Gradle properties, we can everything directly from the `Publication` object.  So, after everything done by the Vanniktech plugin.